modules/apps/headless-apio/media-object/media-object-apio-test/src/testIntegration/java/com/liferay/media/object/apio/internal/architect/resource/test/DefaultMediaObjectNestedCollectionResourceTest.java

```diff
@@ -115,11 +179,13 @@ public void testGetMediaObjectAssetTags() throws Exception {
 
 		List<String> keywords = Arrays.asList("keyword1", "keyword2");
 
+		byte[] bytes = content.getBytes(StandardCharsets.UTF_8);
+
 		FileEntry fileEntry = _addFileEntry(
 			_group.getGroupId(),
 			new MediaObjectImpl(
 				new BinaryFile(
-					new ByteArrayInputStream(content.getBytes(UTF_8)), 0L,
+					new ByteArrayInputStream(bytes), 0L,
```

Comment:
if bytes is used just here. replace it for  content.getBytes(StandardCharsets.UTF_8)